src/openzaak/components/catalogi/admin/besluittype.py

```diff
@@ -20,6 +20,11 @@ class BesluitTypeAdmin(
     list_filter = ("catalogus",)
     search_fields = ("uuid", "omschrijving", "besluitcategorie", "toelichting")
     ordering = ("catalogus", "omschrijving")
+    raw_id_fields = (
+        "catalogus",
+        "zaaktypen",
```

Comment:
Should it be implemented also for M2M relations?

Reply:
@joeribekker in https://github.com/open-zaak/open-zaak/issues/308 you mentioned that `raw_id_fields` should also be used for inlines, so I also added it for the m2m relations, is this correct?